.gitignore

```diff
@@ -7,3 +7,4 @@
 /pkg/
 /spec/reports/
 /tmp/
+/futz/
```

Comment:
Oops, I thought that would be ignored. Eh, it's harmless. It's just a place where I put local test scripts.